### ISSUE TITLE
Did Azure DevOps git repos remove the CORS headers?

### ISSUE THREAD

**Issue by Gaboose** [isomorphic-git/isomorphic-git#1125]
I tried cloning it from the browser like this:
```
git.clone({fs, http, dir, url: 'https://<username>@dev.azure.com/'<username>/<projectname>/_git/<projectname>', onAuth: url => {
    return {
        username: '<username>',
        password: '<password>'
    }
}})
```
And the fetch fails with this error:
```
Access to fetch at 'https://dev.azure.com/<username>/<projectname>/_git/<projectname>/info/refs?service=git-upload-pack' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

It's been discussed on this issue https://github.com/isomorphic-git/isomorphic-git/issues/678, that Azure DevOps had CORS headers, have they been removed or am I connecting wrong?

**Comment by jcubic** [comment]
This is out of scope, please ask questions on [superuser.com](https://superuser.com/questions/ask) or [serverfault.com](https://serverfault.com/questions/ask)
